### PR TITLE
blog: add main text, images

### PR DESCRIPTION
**Pull request by urcades**

### CODE REVIEW

File: content/blog/tools-of-our-own.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
++++
+title = "Tools of Our Own"
+date = 2020-05-14
+description = "What is a digital environment? What does it mean to shape your own digital environment?"
+[extra]
+author = "Matilde Park"
+ship = "~haddef-sigwen"
+image =""
```
Comment (matildepark) [inline content/blog/tools-of-our-own.md]: Needs to be set

File: content/blog/tools-of-our-own.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
++++
+
+<br>
+
+<img class="ba" src="https://media.urbit.org/site/posts/essays/tools-of-our-own-1.png">
+
+## Tools of Our Own
```
Comment (matildepark) [inline content/blog/tools-of-our-own.md]: Cut

**Comment by matildepark** [comment]
You just committed docs and a 20mb zola binary.

**Comment by matildepark** [comment]
You can just do `git reset --soft HEAD~1`, do the commit again and `git push -f` ftr.

**Comment by urcades** [comment]
> You just committed docs and a 20mb zola binary.

Yeah, trying to working around visual studio code's git management interfaces, a lot of its state isn't so clear, will fix

File: content/blog/tools-of-our-own.md (hunk 2 above)
Comment (urcades) [reply content/blog/tools-of-our-own.md]: Are you specifically noting to remove the `## Tools of Our Own`?

Comment (matildepark) [reply content/blog/tools-of-our-own.md]: Yeah, it's not meant as a section title, it's the title title.

**Comment by urcades** [comment]
Pending any further edits, this seems good to go!

**Review by matildepark — APPROVED** [review]
just go ahead and merge it imo

**Pull request merged**